### PR TITLE
[Chuffed] Update from v0.10.4 to v0.13.2

### PR DESCRIPTION
**Pull request by remi-garcia**
Updating Chuffed artifact (necessary for using the SolutionLimit attribute -- x-ref: https://discourse.julialang.org/t/question-about-when-minizinc-will-return-feasible-solutions/116595/8)

### CODE REVIEW

**Comment by remi-garcia** [comment]
Most `-cxx11` pass.

All `-cxx03` fail.

I don't know how to handle that.

**Comment by remi-garcia** [comment]
Only `w64-mingw32` checks fail

**Comment by giordano** [comment]
Error is
```
[13:50:05] /opt/bin/x86_64-w64-mingw32-libgfortran3-cxx11/x86_64-w64-mingw32-g++ --sysroot=/opt/x86_64-w64-mingw32/x86_64-w64-mingw32/sys-root/ -O3 -DNDEBUG -Wl,--whole-archive CMakeFiles/fzn-chuffed.dir/objects.a -Wl,--no-whole-archive -o fzn-chuffed.exe -Wl,--out-implib,libfzn-chuffed.dll.a -Wl,--major-image-version,0,--minor-image-version,0 @CMakeFiles/fzn-chuffed.dir/linklibs.rsp
[13:50:05] libchuffed.a(engine.cpp.obj):engine.cpp:(.text+0x721f): undefined reference to `__imp_WSAStartup'
[13:50:05] libchuffed.a(engine.cpp.obj):engine.cpp:(.text+0x7292): undefined reference to `__imp_getaddrinfo'
[13:50:05] libchuffed.a(engine.cpp.obj):engine.cpp:(.text+0x72be): undefined reference to `__imp_socket'
[13:50:05] libchuffed.a(engine.cpp.obj):engine.cpp:(.text+0x72ca): undefined reference to `__imp_connect'
[13:50:05] libchuffed.a(engine.cpp.obj):engine.cpp:(.text+0x72d1): undefined reference to `__imp_closesocket'
[13:50:05] libchuffed.a(engine.cpp.obj):engine.cpp:(.text+0x73de): undefined reference to `__imp_closesocket'
[13:50:05] libchuffed.a(engine.cpp.obj):engine.cpp:(.text+0x7439): undefined reference to `__imp_freeaddrinfo'
[13:50:05] libchuffed.a(engine.cpp.obj):engine.cpp:(.text+0x7494): undefined reference to `gai_strerrorA'
[13:50:05] libchuffed.a(engine.cpp.obj):engine.cpp:(.text$_ZN10cpprofiler9Connector14sendOverSocketEv[_ZN10cpprofiler9Connector14sendOverSocketEv]+0x18f): undefined reference to `__imp_send'
```
but [`WSAStartup` requires linking to `ws2_32`](https://learn.microsoft.com/en-us/windows/win32/api/winsock/nf-winsock-wsastartup#requirements) (`-lws2_32`) which isn't used here.

**Comment by remi-garcia** [comment]
Is it something we can/should solve with `build_tarballs.jl` or should I open an issue in chuffed repo? https://github.com/chuffed/chuffed

File: C/Chuffed/build_tarballs.jl [hunk 1]
```diff
@@ -2,31 +2,28 @@ using BinaryBuilder, Pkg
...
 cd $WORKSPACE/srcdir/chuffed
-mkdir -p build
-cd build
-cmake \
+cmake -B build \
     -DCMAKE_INSTALL_PREFIX=${prefix} \
     -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
-    -DCMAKE_BUILD_TYPE=Release \
-    ..
+    -DCMAKE_BUILD_TYPE=Release
```
Comment (odow) [inline C/Chuffed/build_tarballs.jl:19]:
```suggestion
    -DCMAKE_BUILD_TYPE=Release \
    -DCP_PROFILER=OFF
```
It looks like it is just coming from the optional profiler. Let's turn that off.

File: C/Chuffed/build_tarballs.jl [hunk 2]
```diff
@@ -2,31 +2,28 @@ using BinaryBuilder, Pkg
...
 # https://github.com/chuffed/chuffed/issues/75
-rm ../chuffed/flatzinc/parser.tab.h
-make -j ${nproc}
-make install
+#rm ../chuffed/flatzinc/parser.tab.h
```
Comment (odow) [inline C/Chuffed/build_tarballs.jl:21]:
```suggestion
```

File: C/Chuffed/build_tarballs.jl [hunk 3]
```diff
@@ -2,31 +2,28 @@ using BinaryBuilder, Pkg
...
-mkdir -p build
-cd build
-cmake \
+cmake -B build \
     -DCMAKE_INSTALL_PREFIX=${prefix} \
     -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
-    -DCMAKE_BUILD_TYPE=Release \
-    ..
+    -DCMAKE_BUILD_TYPE=Release
 # https://github.com/chuffed/chuffed/issues/75
```
Comment (odow) [inline C/Chuffed/build_tarballs.jl:20]:
```suggestion
```

**Comment by remi-garcia** [comment]
Thank you for finding this quick fix @odow, and thank you for the overall help @giordano, tests pass 🎉

**Pull request merged**